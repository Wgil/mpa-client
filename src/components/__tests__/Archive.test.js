import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Archive from '../Archive';

describe('Test Archive.js', () => {
  it('Component Rendered', () => {
    const wrapper = shallow(<Archive />);
    expect(wrapper).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
