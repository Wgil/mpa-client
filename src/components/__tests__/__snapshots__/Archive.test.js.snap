// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Archive.js Component Rendered 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <h1
      className="App-title"
    >
      Welcome to Archive Page
    </h1>
  </header>
</div>
`;
